source/class/qx/bom/Iframe.js

```diff
@@ -73,7 +72,11 @@ qx.Class.define("qx.bom.Iframe",
         }
       }
 
-      return qx.dom.Element.create("iframe", attributes, win);
+      var elem = qx.dom.Element.create("iframe", attributes, win);
+      if (attributes["onload"] == null) {
```

Comment:
This doesn't look right. Are you really checking for `null`? If so, it should use `===` instead of `==`. Are you instead checking for whether `onload` is not in the `attributes` map? In that case, I'd recommend either the specific  `if (! ("onload" in attributes))` or the more generic but oft-used, `if (! attributes["onload"])`. (I'd recommend the former.)

Reply:
Yep, I'll fix it. I was just following form from line 70, but I'll change that too